spearheadblocks/style.scss

```diff
@@ -0,0 +1,34 @@
+/*
+Theme Name: Spearhead Blocks
+Theme URI: https://github.com/automattic/themes/tree/trunk/spearhead-blocks
+Author: Automattic
+Author URI: https://automattic.com
+Description: Spearhead Blocks is the Full Site Editing capable, block based version of the original Spearhead theme.
+Requires at least: 5.8
+Tested up to: 5.9
+Requires PHP: 5.7
+Version: 0.0.1
+License: GNU General Public License v2 or later
+License URI: https://raw.githubusercontent.com/Automattic/themes/trunk/LICENSE
+Template: blockbase
+Text Domain: spearheadblocks
```

Comment:
Convention tells us that this slug should be spearhead-blocks (as well as the folder name, etc)